digital_comms/ccam.py

```diff
@@ -289,15 +415,29 @@ def _small_cell_capacity(self):
 
     @property
     def capacity_margin(self):
+        """obj: The capacity margin in this postcode sector TODO
```

Comment:
"""obj: Capacity margin per postcode sector in Mbps"""

Reply:
Done